containers/tomcat-7/src/main/java/org/mobicents/servlet/sip/startup/SipStandardContext.java

```diff
@@ -252,7 +254,20 @@ protected void prepareServletContext() throws LifecycleException {
 			if(proxyTimerServiceType != null && proxyTimerServiceType.equalsIgnoreCase("Standard")) {
                 proxyTimerService = new ProxyTimerServiceImpl(applicationName);
             } else if(proxyTimerServiceType != null && proxyTimerServiceType.equalsIgnoreCase("Default")) {
-                proxyTimerService = new DefaultProxyTimerService(applicationName);
+            	String strCorePoolSize = this.getServletContext().getInitParameter(TIMER_SERVICE_POOL_SIZE);
+            	if (strCorePoolSize != null && !strCorePoolSize.isEmpty()) {
+            		try {
+            			int CorePoolSize = Integer.parseInt(strCorePoolSize);

```

Comment:
rearrange this logic so if provided size is wrong we start a DefaultProxyTimerService, and we log a WARN log level to the administrator